chromadb/segment/impl/metadata/sqlite.py

```diff
@@ -417,32 +415,8 @@ def _where_map_criterion(
                     for w in cast(Sequence[Where], v)
                 ]
                 clause.append(reduce(lambda x, y: x | y, criteria))
-            elif k == "$in":
```

Comment:
Why can this be removed?

Reply:
It was a leftover from the original design with propositional logic - `{"$in/$nin":{"attr":["a","b","c"]}}`